buildscripts/semantic-convention/templates/SemanticAttributes.java.j2

```diff
@@ -124,6 +124,23 @@ public final class {{class}} {
   @Deprecated
   public static final AttributeKey<String> DB_HBASE_NAMESPACE = stringKey("db.hbase.namespace");
 
+  /**
+   * The size of the uncompressed request payload body after transport decoding. Not set if
+   * transport encoding not used.
+   *
+   * @deprecated  this item has been removed as of 1.13.0 of the semantic conventions. Please use {@link SemanticAttributes#HTTP_REQUEST_CONTENT_LENGTH} instead.
+   */
+  @Deprecated
+  public static final AttributeKey<Long> HTTP_REQUEST_CONTENT_LENGTH_UNCOMPRESSED =
```

Comment:
There's several more of those:
* `HTTP_RESPONSE_CONTENT_LENGTH_UNCOMPRESSED`
* `HTTP_HOST` 
* `NET_PEER_IP`
* `NET_PEER_PORT`
* `NET_HOST_IP`

Can you add these as well?

Reply:
Is our general practice to retain deleted attributes with `@Deprecated` for a single release cycle, indefinitely, some other schedule?

Reply:
Good question -- I think a single release cycle is fine, it lets whoever's using the lib replace those deprecated usages.

Reply:
good catch, although `NET_PEER_PORT` is still here.  updated.

Reply:
Looks like `HTTP_RESPONSE_CONTENT_LENGTH_UNCOMPRESSED` is still missing.

Reply:
whoops. missed request vs. response. added.